server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/stats/ClusterStatsIT.java

```diff
@@ -144,6 +145,28 @@ public void testNodeCounts() {
         }
     }
 
+    // Validate assigning value "master" to setting "node.roles" can get correct count in Node Stats response after MASTER_ROLE deprecated.
+    // TODO: Remove the test after removing MASTER_ROLE.
```

Comment:
Nitpick - I don't think we need these TODOs alongside all of the tests. When we remove `DiscoveryNodeRole.MASTER_ROLE` these tests would automatically break, so we would anyway resolve them then

Reply:
Hi @kartg Thanks for your review! 👍 I will remove the unnecessary TODOs. 
The reason I added them is to giving a clear mind that it definitely can be removed. 😂

Reply:
Removed all the TODO in this PR, and I believe the comment of every newly added test is enough to make people understand they are only used to test the deprecated "master" role setting.